mapcache.cpp

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright (C) 2019 Ricardo Leite. All rights reserved.
+ * Licenced under the MIT licence. See COPYING file in the project root for
+ * details.
+ */
+
+#include "mapcache.h"
+
+// thread cache, uses tsd/tls
+// one cache per thread
+__thread MapCacheBin MapCache;
```

Comment:
nitpick: with the rest of the codebase, this should be `sMapCache`

(same in other places)

Reply:
Fixed.